db/migrate_cards/20180105134606_created_at_for_answers.rb

```diff
@@ -0,0 +1,26 @@
+# -*- encoding : utf-8 -*-
+
+class CreatedAtForAnswers < Card::Migration
+  def up
+    Answer.where(created_at: nil).find_each do |a|
+      next unless a.card
+      if a.card.type_id == Card::MetricValueID
+        begin
+          if a.card.researched?
+            a.update_attributes! created_at: a.card.created_at
+          else
+            a.created_at = a.card.created_at
+            a.answer_id = nil
+            a.card.delete!
+            a.save!
+            puts "successfully migrated calculated answer: #{a.id}"
+          end
+        rescue
+          puts "problem migrating answer: #{a.id} "
```

Comment:
so this basically didn't work.  the cards didn't delete and pretty much every calculated value hit this rescue case.

I didn't think we could hold up the deployment for this, but it's still unresolved.